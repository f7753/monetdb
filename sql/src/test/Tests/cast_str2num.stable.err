stderr of test 'cast_str2num` in directory 'src/test` itself:


# 10:57:40 >  
# 10:57:40 >  Mtimeout -timeout 180 Mserver "--config=/var/tmp/_PREFIX_MONET_GNU_32_d__/etc/monet.conf" --debug=10 --set "monet_mod_path=/var/tmp/_PREFIX_MONET_GNU_32_d__/lib/MonetDB:/var/tmp/_PREFIX_SQL_GNU_32_d__/lib/MonetDB" --dbfarm "/var/tmp/_PREFIX_MONET_GNU_32_d__/var/dbfarm" --set monet_prompt= --trace "--dbname=mTests_src_test" --set sql_port=54321 --set mapi_port=58886 "/var/tmp/_PREFIX_SQL_GNU_32_d__/lib/MonetDB/sqlserver.mil" ; echo ; echo Over..
# 10:57:40 >  

# builtin opt 	gdk_arch = 32biti686-pc-linux-gnu
# builtin opt 	gdk_version = 4.3.9
# builtin opt 	monet_pid = 22340
# builtin opt 	prefix = /var/tmp/_PREFIX_MONET_GNU_32_d__
# builtin opt 	exec_prefix = ${prefix}
# builtin opt 	gdk_dbname = tst
# builtin opt 	gdk_dbfarm = ${prefix}/var/MonetDB
# builtin opt 	gdk_debug = 8
# builtin opt 	gdk_mmap = 32M
# builtin opt 	monet_admin = adm
# builtin opt 	monet_prompt = >
# builtin opt 	monet_welcome = yes
# builtin opt 	monet_mod_path = ${exec_prefix}/lib/MonetDB
# builtin opt 	monet_daemon = yes
# builtin opt 	host = localhost
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_noheaders = no
# builtin opt 	mapi_debug = 0
# builtin opt 	sql_port = 45123
# builtin opt 	sql_debug = 0
# builtin opt 	sql_api = sql(default-schema)
# builtin opt 	sql_user = monetdb
# builtin opt 	sql_schema = default-schema
# builtin opt 	sql_logdir = ${prefix}/var/MonetDB/log
# config opt 	prefix = /var/tmp/_PREFIX_MONET_GNU_32_d__
# config opt 	exec_prefix = ${prefix}
# config opt 	gdk_dbfarm = ${prefix}/var/dbfarm
# config opt 	gdk_dbname = demo
# config opt 	gdk_debug = 0
# config opt 	gdk_mmap = 33554432
# config opt 	monet_admin = adm
# config opt 	monet_prompt = monet>
# config opt 	monet_welcome = yes
# config opt 	monet_mod_path = ${exec_prefix}/lib/MonetDB
# config opt 	monet_daemon = no
# config opt 	sql_logdir = ${prefix}/var/dbfarm/log
# cmdline opt 	gdk_debug = 10
# cmdline opt 	monet_mod_path = /var/tmp/_PREFIX_MONET_GNU_32_d__/lib/MonetDB:/var/tmp/_PREFIX_SQL_GNU_32_d__/lib/MonetDB
# cmdline opt 	gdk_dbfarm = /var/tmp/_PREFIX_MONET_GNU_32_d__/var/dbfarm
# cmdline opt 	monet_prompt = 
# cmdline opt 	gdk_dbname = mTests_src_test
# cmdline opt 	sql_port = 54321
# cmdline opt 	mapi_port = 58886

# 10:57:41 >  
# 10:57:41 >  Mtimeout -timeout 60 Msql -u monetdb -P monetdb --host=draco --port=54321 < cast_str2num.sql
# 10:57:41 >  


# 10:57:41 >  
# 10:57:41 >  Done.
# 10:57:41 >  

