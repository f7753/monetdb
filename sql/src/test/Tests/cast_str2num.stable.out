stdout of test 'cast_str2num` in directory 'src/test` itself:


# 10:57:40 >  
# 10:57:40 >  Mtimeout -timeout 180 Mserver "--config=/var/tmp/_PREFIX_MONET_GNU_32_d__/etc/monet.conf" --debug=10 --set "monet_mod_path=/var/tmp/_PREFIX_MONET_GNU_32_d__/lib/MonetDB:/var/tmp/_PREFIX_SQL_GNU_32_d__/lib/MonetDB" --dbfarm "/var/tmp/_PREFIX_MONET_GNU_32_d__/var/dbfarm" --set monet_prompt= --trace "--dbname=mTests_src_test" --set sql_port=54321 --set mapi_port=58886 "/var/tmp/_PREFIX_SQL_GNU_32_d__/lib/MonetDB/sqlserver.mil" ; echo ; echo Over..
# 10:57:40 >  

# Monet Database Server V4.3.9
# Copyright (c) 1993-2003, CWI. All rights reserved.
# compiled for i686-pc-linux-gnu/32bit; dynamically linked.
printf("\nReady.\n");

Ready.
quit();

Over..

# 10:57:41 >  
# 10:57:41 >  Mtimeout -timeout 60 Msql -u monetdb -P monetdb --host=draco --port=54321 < cast_str2num.sql
# 10:57:41 >  

SQL  connected to database mTests_src_test using schema sys
#     1: 1 create table T ( I int, F float ) ;
#     2: 0 
#     3: 1 insert into T values ( 1, 0.1 );
SQL  no Rows affected
#     4: 1 insert into T values ( 2, 0.2 );
SQL  no Rows affected
#     5: 1 insert into T values ( 3, 0.3 );
SQL  no Rows affected
#     6: 0 
#     7: 1 select * from T where I <  2;
#---------
#| i | f |
#---------
[ 1,	0.10000000000000001	]
SQL  1 Row affected
#     8: 1 select * from T where I <  '2';
#---------
#| i | f |
#---------
[ 1,	0.10000000000000001	]
SQL  1 Row affected
#    10: 1 select * from T where F <  0.2;
#---------
#| i | f |
#---------
[ 1,	0.10000000000000001	]
SQL  1 Row affected
#    11: 1 select * from T where F <  '0.2';
#---------
#| i | f |
#---------
[ 1,	0.10000000000000001	]
SQL  1 Row affected
#    13: 0 
#    14: 1 select * from T where I <= 2;
#---------
#| i | f |
#---------
[ 1,	0.10000000000000001	]
[ 2,	0.20000000000000001	]
SQL  2 Rows affected
#    15: 1 select * from T where I <= '2';
#---------
#| i | f |
#---------
[ 1,	0.10000000000000001	]
[ 2,	0.20000000000000001	]
SQL  2 Rows affected
#    17: 1 select * from T where F <= 0.2;
#---------
#| i | f |
#---------
[ 1,	0.10000000000000001	]
[ 2,	0.20000000000000001	]
SQL  2 Rows affected
#    18: 1 select * from T where F <= '0.2';
#---------
#| i | f |
#---------
[ 1,	0.10000000000000001	]
[ 2,	0.20000000000000001	]
SQL  2 Rows affected
#    20: 0 
#    21: 1 select * from T where I  = 2;
#---------
#| i | f |
#---------
[ 2,	0.20000000000000001	]
SQL  1 Row affected
#    22: 1 select * from T where I  = '2';
#---------
#| i | f |
#---------
[ 2,	0.20000000000000001	]
SQL  1 Row affected
#    24: 1 select * from T where F  = 0.2;
#---------
#| i | f |
#---------
[ 2,	0.20000000000000001	]
SQL  1 Row affected
#    25: 1 select * from T where F  = '0.2';
#---------
#| i | f |
#---------
[ 2,	0.20000000000000001	]
SQL  1 Row affected
#    27: 0 
#    28: 1 select * from T where I >= 2;
#---------
#| i | f |
#---------
[ 2,	0.20000000000000001	]
[ 3,	0.29999999999999999	]
SQL  2 Rows affected
#    29: 1 select * from T where I >= '2';
#---------
#| i | f |
#---------
[ 2,	0.20000000000000001	]
[ 3,	0.29999999999999999	]
SQL  2 Rows affected
#    31: 1 select * from T where F >= 0.2;
#---------
#| i | f |
#---------
[ 2,	0.20000000000000001	]
[ 3,	0.29999999999999999	]
SQL  2 Rows affected
#    32: 1 select * from T where F >= '0.2';
#---------
#| i | f |
#---------
[ 2,	0.20000000000000001	]
[ 3,	0.29999999999999999	]
SQL  2 Rows affected
#    34: 0 
#    35: 1 select * from T where I >  2;
#---------
#| i | f |
#---------
[ 3,	0.29999999999999999	]
SQL  1 Row affected
#    36: 1 select * from T where I >  '2';
#---------
#| i | f |
#---------
[ 3,	0.29999999999999999	]
SQL  1 Row affected
#    38: 1 select * from T where F >  0.2;
#---------
#| i | f |
#---------
[ 3,	0.29999999999999999	]
SQL  1 Row affected
#    39: 1 select * from T where F >  '0.2';
#---------
#| i | f |
#---------
[ 3,	0.29999999999999999	]
SQL  1 Row affected
#    41: 0 
#    42: 1 drop table T;

# 10:57:41 >  
# 10:57:41 >  Done.
# 10:57:41 >  

